unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NotificationMessagesHUD/ChatNotificationController.cs

```diff
@@ -0,0 +1,55 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using DCL.Interface;
+using DCL;
+
+public class ChatNotificationController : IHUD
+{
+    IChatController chatController;
+    MainChatNotificationsComponentView view;
+    private IUserProfileBridge userProfileBridge;
+    internal BaseVariable<Transform> isInitialized => DataStore.i.HUDs.isNotificationPanelInitialized;
+
+    private UserProfile ownUserProfile;
+
+    public ChatNotificationController(IChatController chatController)
+    {
+        this.chatController = chatController;
+    }
+
+    public void Initialize(IUserProfileBridge userProfileBridge, MainChatNotificationsComponentView view)
+    {
+        this.userProfileBridge = userProfileBridge;
+        this.view = view;
+        ownUserProfile = userProfileBridge.GetOwn();
+        chatController.OnAddMessage += HandleMessageAdded;
+    }
```

Comment:
Why are you separating the injection of dependencies in 2 methods? I think we could do it directly in the constructor and delete the `Initialize` method.

Reply:
Done